_pages/rentals/cabins.md

```diff
@@ -30,6 +30,8 @@ The exclusive rental rate for Camelot is $300 per night, and the exclusive renta
 
 We do not permit exclusive rentals during certain weekends. For example, both cabins are used by MITOC's winter school program for every weekend in January. Always contact the cabin managers before you submit payment for a reservation.
 
+We emphasize again: RESERVATION FEES ARE NOT REFUNDABLE. 
```

Comment:
I can link a resource later, but for accessibility reasons one should generally use the emphasis tag (`<em>`) for emphasis versus using all caps:
```suggestion
We emphasize again: <em>reservation fees are not refundable.</em>

Reply:
While you *can* use HTML in this Markdown file, you can also just use italics and Markdown should convert to `<em>` when generating HTML:
```suggestion
We emphasize again: *reservation fees are not refundable.*